composer.json

```diff
@@ -17,12 +17,11 @@
 	"require": {
 		"php": "^7.1.3",
 		"ext-json": "*",
-		"illuminate/support": "~5.8.0|~5.9.0",
-		"illuminate/contracts": "~5.8.0|~5.9.0"
+		"illuminate/support": "~5.8.0"
```

Comment:
I'd like to keep ~5.9.0 in here. Removing contracts and keeping support is fine.